README.md

```diff
@@ -245,6 +245,9 @@ Includes the content of a file or a group of files.
 - <a name="include_encoding" href="#include_encoding">#</a>
   **encoding** (_utf-8_): Specify the encoding of the included file.
   If not defined `utf-8` will be used.
+  **recursive** (_true_): When this option is disabled, it will not process
+  included files recursively. This can be useful when you want to include complex
+  HTML contents. Possible values are `true` and `false`.
```

Comment:
```suggestion
  included files recursively. Possible values are `true` and `false`.
```

I think that the statement about HTML is confusing.

Reply:
ok, i remove that as well then